res/layout/share_file_layout.xml

```diff
@@ -99,7 +93,31 @@
                 android:layout_height="wrap_content"
                 android:layout_gravity="start"
                 android:text="@string/share_with_user_section_title"
-                android:textColor="@color/color_accent"/>
+                android:textColor="@color/color_accent" />
+
+            <LinearLayout
```

Comment:
@davigonz , with this change the 'add user or groups' button is shown over the list of shared users, instead of below. Was this on purpose?